server/data/welcomeClient.test.ts

```diff
@@ -51,52 +52,44 @@ describe('welcomeClient', () => {
   describe('getExpectedArrivals', () => {
     const activeCaseLoadId = 'MDI'
     const date = moment()
-    const expectedArrivals: Arrival[] = []
+    const arrivals: Arrival[] = []
     it('should return data from api', async () => {
       fakeWelcomeApi
         .get(`/prisons/${activeCaseLoadId}/arrivals?date=${date.format('YYYY-MM-DD')}`)
         .matchHeader('authorization', `Bearer ${token}`)
-        .reply(200, expectedArrivals)
+        .reply(200, arrivals)
 
       const output = await welcomeClient.getExpectedArrivals(activeCaseLoadId, date)
-      expect(output).toEqual(expectedArrivals)
+      expect(output).toEqual(arrivals)
     })
   })
 
   describe('getTransfers', () => {
     const activeCaseLoadId = 'MDI'
-    const expectedArrivals: Arrival[] = []
+    const transfers: Arrival[] = []
     it('should return data from api', async () => {
       fakeWelcomeApi
         .get(`/prisons/${activeCaseLoadId}/transfers/enroute`)
         .matchHeader('authorization', `Bearer ${token}`)
-        .reply(200, expectedArrivals)
+        .reply(200, transfers)
 
       const output = await welcomeClient.getTransfers(activeCaseLoadId)
-      expect(output).toEqual(expectedArrivals)
+      expect(output).toEqual(transfers)
     })
   })
 
   describe('get Transfer', () => {
     const activeCaseLoadId = 'MDI'
     const prisonNumber = 'A1234AB'
-    const expectedArrival: Transfer = {
-      firstName: 'Sam',
-      lastName: 'Smith',
-      dateOfBirth: '1971-02-01',
-      prisonNumber: 'A1234AA',
-      pncNumber: '01/1234X',
-      date: '2020-02-23',
-      fromLocation: 'Kingston-upon-Hull Crown Court',
-    }
```

Comment:
for instance - these tests don't require the transfer to have any specific values - it just requires a valid one.

We can completely change all the values with out breaking the test.

We want to limit  the coupling of test data in tests  - if we change the default values in the factories then it should break few if any tests